runtime/interpreter/value.go

```diff
@@ -19952,13 +19715,15 @@ func NewUnmeteredEphemeralReferenceValue(
 }
 
 func NewEphemeralReferenceValue(
-	gauge common.MemoryGauge,
+	interpreter *Interpreter,
 	authorized bool,
 	value Value,
 	borrowedType sema.Type,
 ) *EphemeralReferenceValue {
-	common.UseMemory(gauge, common.EphemeralReferenceValueMemoryUsage)
-	return NewUnmeteredEphemeralReferenceValue(authorized, value, borrowedType)
+	common.UseMemory(interpreter, common.EphemeralReferenceValueMemoryUsage)
+	ref := NewUnmeteredEphemeralReferenceValue(authorized, value, borrowedType)
```

Comment:
tat's a good flag. The `NewUnmeteredEphemeralReferenceValue` is currently mostly (only) used for tests, but think I will just move the tracking to the `NewUnmeteredEphemeralReferenceValue` so even in the future if this was used in a non-test use-case it would be safe.